src/lib/Form/Type/Search/SimpleSearchType.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\EzPlatformAdminUi\Form\Type\Search;
+
+use EzSystems\EzPlatformAdminUi\Form\Data\Search\SimpleSearchData;
+use Symfony\Component\Form\Extension\Core\Type\HiddenType;
+use Symfony\Component\Form\Extension\Core\Type\SearchType;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class SimpleSearchType extends AbstractType
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder->setMethod('GET')
+            ->add('query', SearchType::class)
+            ->add('page', HiddenType::class)
+            ->add('limit', HiddenType::class)
+        ;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configureOptions(OptionsResolver $resolver)
+    {
+        $resolver->setDefaults([
+            'data_class' => SimpleSearchData::class,
+            'csrf_protection' => false,
+        ]);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getBlockPrefix()
+    {
+        return null;
```

Comment:
Why do you set it to `null`? Block prefix is crucial for form theming. We do not have any strong convention but I started using underscored class name with type replaced with form word: `simple_search_form`.

Reply:
I used it because I want simple parameters in the URL. I don't think so the block prefix is crucial for form theming, rather id or class added in twig template. I think form builder is not a good place for styling. What do you think?

Reply:
Hmm, I'm not sure if you understand my point. All forms should have unique block name, this allows you to do overrides in twig:

```twig
{% block simple_search_form %}
    {% set attr = attr|merge({'class': 'custom-class'}) %}
    {{ block('form_widget') }}
{% endblock %}
```

Which is what we do in order to separate styling information (like `class` HTML attribute) from PHP code.